content/en/docs/v3.6/dev-guide/grpc_naming.md

```diff
@@ -44,6 +44,14 @@ The etcd client's `endpoints.Manager` method can also register new endpoints wit
 em := endpoints.NewManager(client, "foo/bar/my-service")
 err := em.AddEndpoint(context.TODO(),"foo/bar/my-service/e1", endpoints.Endpoint{Addr:"1.2.3.4"});
 ```
+To enable round-robin load balancing when dialing service with multiple endpoints, you can set up you connection with grpc
+ internal round-robin load balancer:
+
+ ```go
+
+ 	conn, gerr := grpc.Dial("etcd:///foo", grpc.WithResolvers(etcdResolver),
```

Comment:
Actually Resolver is an experimental gRPC API, and we need to remove the dependency on it. FYI. https://github.com/etcd-io/etcd/issues/15145

I am OK to merge this PR firstly, and update it when https://github.com/etcd-io/etcd/issues/15145 is done. 

cc @spzala

Reply:
@ahrtr thanks and that sounds good to me too!